packages/wdio-browserstack-service/src/request-handler.ts

```diff
@@ -67,7 +67,9 @@ export default class RequestQueueHandler {
 
     callCallback = async (data: UploadType[], kind: string) => {
         BStackLogger.debug('calling callback with kind ' + kind)
-        this.callback && await this.callback(data)
+        if (this.callback) {
+            await this.callback(data)
```

Comment:
Instead of doing the if check you can remove the if check and change this line.
This only executes the function if it exists (when assigned/returned it will be undefined)
```suggestion
            await this.callback?.(data)
```